jetson/nucleo_bridge/Controller.cpp

```diff
@@ -102,7 +102,9 @@ void Controller::refresh_calibration_data()
 
         transact(CALIBRATED, nullptr, UINT8_POINTER_T(&raw_calibration_data));
 
-        calibrated = raw_calibration_data == 0xF;
+        // raw_calibration_data is either 0xFF or 0x00.
+        // 0xFF means it is calibrated, 0x00 means not calibrated.
+        calibrated = raw_calibration_data != NOT_CALIBRATED_BOOL;

```

Comment:
nitpick, raw_calibration_data --> calibration_state? 
NOT_CALIBRATED_BOOL --> #define CALIBRATED 0xFF
then calibrated = (calibration_state == CALIBRATED)

Reply:
the problem with #define CALIBRATED 0xFF is that for some odd reason it didn't work when I tried it... I can try to test it again eventually but ya I agree that this would look better

Reply:
strange indeed..